src/pixi-multistyle-text.ts

```diff
@@ -1,4 +1,4 @@
-/// <reference types="pixi.js" />
+import * as PIXI from "pixi.js";
```

Comment:
I wonder if we can import only what is used in the file tho. But it's ok for now.
Thanks for doing this. I'll release a new release as 0.10.0 to make sure this doesn't break for any other users currently using 0.9